bokeh/models/callbacks.py

```diff
@@ -38,16 +38,13 @@ def from_py_func(cls, func):
         try:
             from flexx.pyscript import py2js
         except ImportError:
-            if sys.version_info < (3, ):
```

Comment:
Let's be consistent and use this everywhere to manage warning about imports that are required for optional features: https://github.com/bokeh/bokeh/blob/master/bokeh/util/dependencies.py#L23